src/main/java/session_17_equals_hashcode_generics/challenge/challenge_10/AppInformation.java

```diff
@@ -0,0 +1,40 @@
+package session_17_equals_hashcode_generics.challenge.challenge_10;
+
+public class AppInformation {
+    private String name;
+    private long id;
+    private String dateOfLastUpdate;
```

Comment:
When we work with date & time values, we want to use the default classes that can handle this specific types, like LocalDate, LocalDateTime, etc.